include/api.php

```diff
@@ -252,8 +252,8 @@ function api_check_method($method) {
 	 * @param App $a
 	 * @return string API call result
 	 */
-	function api_call(App &$a){
-		global $API, $called_api;
+	function api_call(&$a){
+		GLOBAL $API, $called_api;
```

Comment:
Why did you change that almost perfectly formatted declaration?

It still lacks a space before the opening brace, but removing the class name in the parameter list and uppercasing the `global` keyword weren't really warranted.